app/src/main/res/layout/activity_register.xml

```diff
@@ -28,7 +28,7 @@
         <EditText
             android:id="@+id/editTextEmail"
             android:layout_width="match_parent"
-            android:hint="Email Address"
+            android:hint="@string/email_address"
```

Comment:
Suggest naming all these hints more descriptively; we want to ensure hints and fields are properly abstracted so we can keep them separate from other places that might use the text "Email Address" (to allow changing of hints and strings separately)

Suggest something like "hint_email_address"